### ISSUE TITLE
build tests are failing

### ISSUE THREAD

**Issue by jdwtf** [kubernetes-sigs/aws-efs-csi-driver#1456]
/kind bug

**What happened?**
[this commit](https://github.com/kubernetes-sigs/aws-efs-csi-driver/commit/d21d258379e12bf7200ca5c2697a60ac31fed571) changed the rust and cargo installation during the docker build from yum to a dynamic method using rustup. This is causing some tests to fail as the rpm make for efs-utils expects rust and cargo to be visible from a yum install. Example of failures from pull requests can be found [here](https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1455) and [here](https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1451).

**What you expected to happen?**
The testing to pass.

**How to reproduce it (as minimally and precisely as possible)?**
Simplest way is to retest either of those PRs above and see Prow output.

**Anything else we need to know?**:

**Environment**
- Kubernetes version (use `kubectl version`): N/A
- Driver version: N/A

**Please also attach debug logs to help us better diagnose**
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_aws-efs-csi-driver/1455/pull-aws-efs-csi-driver-e2e/1836084250353340416

- Instructions to gather debug logs can be found [here](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/troubleshooting)

**Comment by jdwtf** [comment]
Example error message:
```
#12 47.18 	cargo is needed by amazon-efs-utils-2.0.4-2.amzn2.x86_64
#12 47.18 	rust is needed by amazon-efs-utils-2.0.4-2.amzn2.x86_64 
```

**Comment by mskanth972** [comment]
Fixed in the latest version v2.0.8
[PR](https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1455)
